pkg/acsengine/pki.go

```diff
@@ -117,8 +117,6 @@ func createCertificate(commonName string, caCertificate *x509.Certificate, caPri
 		template.KeyUsage |= x509.KeyUsageCertSign
 		template.IsCA = isCA
 	} else if isServer {
-		extraIPs = append(extraIPs, net.ParseIP("10.0.0.1"))
```

Comment:
because it's done here: https://github.com/Azure/acs-engine/pull/473/files#diff-8edb2653431a5da9ad1f4286097702a7R224

Again, otherwise this wouldn't work in the first place.